control-plane-operator/controllers/hostedcontrolplane/v2/kas/kms/azure.go

```diff
@@ -242,3 +246,9 @@ func buildVolumeKMSSecretStore(v *corev1.Volume) {
 		},
 	}
 }
+
+func AdaptAzureSecretProvider(cpContext component.WorkloadContext, secretProvider *secretsstorev1.SecretProviderClass) error {
+	managedIdentity := cpContext.HCP.Spec.SecretEncryption.KMS.Azure.KMS
+	secretproviderclass.ReconcileManagedAzureSecretProviderClass(secretProvider, cpContext.HCP, managedIdentity)
```

Comment:
```suggestion
	secretproviderclass.ReconcileManagedAzureSecretProviderClass(secretProvider, cpContext.HCP, managedIdentity, true)
```
forgot to pass `true` here :)